builtin/game/chat.lua

```diff
@@ -85,9 +87,10 @@ core.register_on_chat_message(function(name, message)
 			core.chat_send_player(name, result)
 		end
 	else
-		core.chat_send_player(name, "You don't have permission"
-				.. " to run this command (missing privileges: "
-				.. table.concat(missing_privs, ", ") .. ")")
+		core.chat_send_player(name,
+				S("You don't have permission to run this command "
```

Comment:
reuse the string please

```suggestion
				S("You don't have permission to run this command "
				.. "(missing privileges: @1)."
```